JetMETCorrections/Modules/plugins/JetResolutionDBWriter.cc

```diff
@@ -46,7 +46,7 @@ void JetResolutionDBWriter::beginJob() {
   if (s.isAvailable()) {
     std::cout << "Setting up payload record " << m_record << std::endl;
     cond::Time_t sinceTime = s->isNewTagRequest(m_record) ? s->beginOfTime() : s->currentTime();
-    s->writeOne<JME::JetResolutionObject>(jerObject, sinceTime, m_record);
+    s->writeOneIOV<JME::JetResolutionObject>(jerObject, sinceTime, m_record);
```

Comment:
```suggestion
    s->writeOneIOV(jerObject, sinceTime, m_record);
```
This would also work